src/button/component.jsx

```diff
@@ -425,6 +435,15 @@ export let Button : Component<ButtonOptions> = create({
                 return {};
             },
             decorate({ allowed = [], disallowed = [] } : Object = {}, props : ButtonOptions) : {} {
+                
+                fundingLogoThrottle = buildFundingLogoThrottle({ ...normalizeProps(props), browserLocale: getBrowserLocale() });
+                if (fundingLogoThrottle) {
+                    allowed = [ ...allowed, FUNDING.CREDIT ];
+                    const creditIndex = disallowed.indexOf(FUNDING.CREDIT);
```

Comment:
Lot of iteration going on here. Can we just do `disallowed = disallowed.filter(source => (source !== FUNDING.CREDIT))`?

Reply:
Actually, thinking about it -- if a merchant has already opted-out of credit, do we really want to disregard that opt-out?

Reply:
Yup/